### PR TITLE
Forced fluff to compile with dwlib

### PR DESCRIPTION
**Pull request by brainface**
There's... there's a few errors? warnings? Compiling c ain't my forte man
you want this to work this'll take your logic probes.

### CODE REVIEW

File: src/packages/dwlib/dwlib.cc [hunk 1]
```diff
@@ -42,6 +45,9 @@ void query_multiple_short(svalue_t *arg, const char *type, int no_dollars, int q
   object_t *ob;
   char *fun;
 
+  //mdw--old code dern't work, copying this from elsewhere.
+  auto max_string_length = CONFIG_INT(__MAX_STRING_LENGTH__);
```
Comment (FractalBobz) [inline src/packages/dwlib/dwlib.cc:49]: I'm assuming by changing MAX STRING LENGTH we can change the max length of posts?
Or is this a base C variable? 
I'll look into this as kitty wants bigger posts.

**Comment by FractalBobz** [comment]
All looks super good. Thanks ^_^

File: src/packages/dwlib/dwlib.cc (hunk 1 above)
Comment (brainface) [inline src/packages/dwlib/dwlib.cc:49]: That's actually a variable set in some other config file that the DW lib
needed to reference--i assume we can change it affecting the whole lib but
there may also be c limitations as you say? (And if so we could work around
it by splitting things up and such?)

Comment (brainface) [inline src/packages/dwlib/dwlib.cc:49]: also i made a fringe repository that excludes um... driver files and stuff
and just has the lib. (maybe... on second thought... you might actually
/want/ the driver stuff

essentially what i'm working on is just:
http://lpmuds.net/files/discworld/dw_fluffos_v3.zip
unzipped and fluff compiled. as i do whatevs i'll git commit etc et al?

**Pull request merged**